docs/serpen_aqua_ubi.md

```
@@ -0,0 +1,256 @@
+   ### Installing Serpen via Aqua
+
+   If you use [aquaproj/aqua](https://aquaproj.github.io/), you can install Serpen by adding it to your Aqua registry config. For example, after Serpen is added to Aqua’s Standard Registry, include in your `aqua.yaml`:
+
+   ```yaml
+   registries:
+     - type: standard
+       ref: latest # uses the latest standard registry
+   packages:
+     - name: <GitHub-Owner>/serpen@<version>
+   ```
+   ````
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)